components/event-block/EventBlockText.vue

```diff
@@ -11,37 +11,51 @@
     <h3
       class="event-block-text__title"
       :class="{
-        'event-block-text__title--orange': props.titleColor === 'orange',
-        'event-block-text__title--blue': props.titleColor === 'blue',
+        'event-block-text__title--orange': titleColor === 'orange',
+        'event-block-text__title--blue': titleColor === 'blue',
       }"
     >
-      {{ props.title }}
+      {{ title }}
     </h3>
+    <p
+      v-if="subtitle"
+      class="event-block-text__subtitle"
+    >
+      {{ subtitle }}
+    </p>
     <div
       class="event-block-text__copy"
-      v-html="props.body"
+      v-html="htmlBody"
     />
     <p
-      v-if="props.callToActionLabel && props.callToActionUrl"
+      v-if="callToActionLabel && callToActionUrl"
       class="event-block-text__cta"
     >
-      <a :href="props.callToActionUrl">
-        {{ props.callToActionLabel }}
+      <a :href="callToActionUrl">
+        {{ callToActionLabel }}
       </a>
     </p>
   </div>
 </template>
 
 <script>
+  import renderMarkedContent from '~/lib/custom-marked';
+
   export default {
     props: {
       showWaveMarker: Boolean,
       title: String,
       titleColor: String,
+      subtitle: String,
       body: String,
       callToActionLabel: String,
       callToActionUrl: String,
     },
+    computed: {
+      htmlBody () {
+        return renderMarkedContent(this.body);
```

Comment:
this logic could be achieved by setting `body(markdown: true)` to let Dato to the transform and then use CSS to highlight the first paragraph